lib/node_modules/@stdlib/stats/base/dists/levy/mode/benchmark/benchmark.native.js

```diff
@@ -51,8 +51,8 @@ bench( pkg+'::native', opts, function benchmark( b ) {
 	mu = new Float64Array( len );
 	c = new Float64Array( len );
 	for ( i = 0; i < len; i++ ) {
-		mu[ i ] = ( randu() * 100.0 ) - 50.0;
-		c[ i ] = ( randu() * 20.0 ) + EPS;
+		mu[ i ] = ( uniform( EPS, 10.0 ) );
```

Comment:
```suggestion
		mu[ i ] = uniform( -50.0, 50.0 );
```

Avoid unnecessary parentheses.